### DISCUSSION TITLE
why use strong names

### DISCUSSION THREAD

**Discussion by drewburlingame** [SimonCropp/MarkdownSnippets#438] [General]
Hey Simon, I notced all the projects are strong named and was curious why.  Afaik, there's no benefit to it with dotnet core.  Was it required for the msbuild task?

**Comment by SimonCropp** [comment]
nope, u r correct, they dont need to be strong named. This project also produces a library nuget, so when i strong named that i just did it at the solution level.

**Reply by drewburlingame** [reply]
Out of curiosity, why did you strong name the library nuget?  Was there a requirement for a consumer of the nuget?

**Reply by SimonCropp** [reply]
because if you dont, you get spammed by people who dont understand strong naming and want it. it is best not to fight them

**Reply by drewburlingame** [reply]
How annoying.  Thanks for the insight.
